airflow/www/utils.py

```diff
@@ -374,6 +375,30 @@ def get_chart_height(dag):
     return 600 + len(dag.tasks) * 10
 
 
+def get_python_source(x):
```

Comment:
Please add type annotations for public functions

Reply:
Added get_python_source(x: Union[Callable, str]) -> str